tachyon/zk/plonk/circuit/region.h

```diff
@@ -98,6 +98,91 @@ class Region {
       return Error::kNone;
     }
   };
+
+  explicit Region(Layouter* layouter) : layouter_(layouter) {}
+
+  // See the comment above.
+  Error EnableSelector(AnnotateCallback annotate, const Selector& selector,
+                       size_t offset) {
+    return layouter_->EnableSelector(std::move(annotate), selector, offset);
+  }
+
+  // See the comment above.
+  void NameColumn(AnnotateCallback annotate, const AnyColumn& column) {
```

Comment:
```suggestion
  void NameColumn(AnnotateCallback annotate, const AnyColumn& column) override {
```
Plus, if we get to add override keyword, how about removing " // See the comment above." comment?